subsys/net/ip/net_shell.c

```diff
@@ -2790,9 +2830,24 @@ static int cmd_net_ping(const struct shell *shell, size_t argc, char *argv[])
 	char *host;
 	int ret;
 
-	ARG_UNUSED(argc);
+	unsigned int count = 3;
+	unsigned int interval = 1000;
+
+	for (size_t i = 1; i < argc - 1; ++i) {
+		switch (*argv[i]) {
+		case 'c':
```

Comment:
I don't see commit message mentioning that new "ping" command options are added. Neither help text seems to be updated.

And as far as I can tell, the command would look like `ping c5 i1 host`. Why don't we make it truly similar to unix commands and be `ping -c5 -i1 host`.

Reply:
I agree with @pfalcon here, use -c and -i for options, it might be also good idea to allow a space character between option and the value.

Reply:
Ok, I initially wanted to avoid adding a dependency on libc or re-implementing `getopt()` poorly, but now have opted for the latter.